main.go

```diff
@@ -29,3 +31,12 @@ func main() {
 		panic(err)
 	}
 }
+
+// checkNetIsolation checks if a loopback exempt is in place to allow the
+// hosting device to join the server. This is only relevant on Windows.
+func checkNetIsolation() {
+	if runtime.GOOS != "windows" {
+		return
+	}
+	data, _ := exec.Command("CheckNetIsolation", "LoopbackExempt", "-s", `-n="microsoft.minecraftuwp_8wekyb3d8bbwe"`).CombinedOutput()
```

Comment:
Fairly certain this has to be run in Admin PowerShell. Did you test this?

Reply:
Instead of having it automatically run, why not give a warning message? That way tedac won't have to be ran as Admin

Reply:
That also works.